babel.config.js

```diff
@@ -3,5 +3,6 @@ module.exports = function( api ) {
 
 	return {
 		presets: [ '@wordpress/babel-preset-default' ],
+		plugins: [ 'babel-plugin-inline-json-import' ],
```

Comment:
Is this something that can be useful for our default preset?

Reply:
Yes, I think it would land there eventually once proved that we like this approach 👍 

By the way, this plugin still depends on Babel 6. Not sure it's an issue, just noting.